### PR TITLE
notebook on using bdbag to retrieve datasets

### PR DESCRIPTION
**Pull request by rpwagner**

### CODE REVIEW

File: 3_create_markdown_pages.ipynb
Comment (zonca) [inline 3_create_markdown_pages.ipynb:1]: had renamed this already

File: 4_jsonld_bdbag.ipynb
Comment (zonca) [inline 4_jsonld_bdbag.ipynb:1]: can you add this to https://github.com/cheapandfair/cheapandfair-gateways-2024/blob/main/4_test_portal.ipynb?

**Review by zonca — CHANGES_REQUESTED** [review]

File: 3_create_markdown_pages.ipynb
Comment (rpwagner) [reply 3_create_markdown_pages.ipynb:1]: ah, that explains the conflict!

File: 4_jsonld_bdbag.ipynb
Comment (rpwagner) [reply 4_jsonld_bdbag.ipynb:1]: Absolutely

**Pull request closed without merging**